encode.cpp

```diff
@@ -0,0 +1,23 @@
+// Data Encryption System: Rahul wants to send encoded messages to his friend Ram. 
+// Develop an algorithm to encode all the digits, special characters, lower and upper case alphabets.
+
+
+#include<iostream>
+#include<string>
+
+using namespace std;
+int main(){
+    int i;
```

Comment:
move the code to a method and call the method in main

Reply:
I have made the changes. please review :)